.NET/Microsoft.Recognizers.Text.Sequence/Constants.cs

```diff
@@ -37,16 +37,22 @@ public static class Constants
 
         public const string PHONE_NUMBER_REGEX_GENERAL= "GeneralPhoneNumber";
 
-        public const string PHONE_NUMBER_REGEX_BRAZIL = "BrazilPhoneNumber";
+        public const string PHONE_NUMBER_REGEX_BR = "BRPhoneNumber";
 
         public const string PHONE_NUMBER_REGEX_UK = "UKPhoneNumber";
 
-        public const string PHONE_NUMBER_REGEX_GERMANY = "GermanyPhoneNumber";
+        public const string PHONE_NUMBER_REGEX_DE = "DEPhoneNumber";
 
         public const string PHONE_NUMBER_REGEX_US = "USPhoneNumber";
 
         public const string PHONE_NUMBER_REGEX_CN = "CNPhoneNumber";
-        
+
+        public const string PHONE_NUMBER_REGEX_DK = "DKPhoneNumber";
+
+        public const string PHONE_NUMBER_REGEX_IT = "ITPhoneNumber";
+
+        public const string PHONE_NUMBER_REGEX_NL = "NLlandsPhoneNumber";
```

Comment:
`NLPhoneNumber` would be fine for the Netherlands. (instead of `NLlandsPhoneNumber`)

Reply:
Yeah, sorry, that is a mistake, I will fix that, thanks.

Reply:
Our model work well with these two cases and the cases have been added to cases, thanks for providing them!